package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "pouchdb-mapreduce",
-  "version": "2.3.2",
-  "description": "Map Reduce =====",
+  "version": "2.3.3",
+  "description": "PouchDB map/reduce",
```

Comment:
Npm init grabs the first paragraph of the readme as a default description

On Wed, Apr 15, 2015, 8:56 AM Nolan Lawson <notifications@github.com> wrote:

> In package.json
> <https://github.com/pouchdb/mapreduce/pull/268#discussion_r28416275>:
>
> > @@ -1,7 +1,7 @@
> >  {
> >    "name": "pouchdb-mapreduce",
> > -  "version": "2.3.2",
> > -  "description": "Map Reduce =====",
> > +  "version": "2.3.3",
> > +  "description": "PouchDB map/reduce",
>
> Not sure what the point of this ==== was.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pouchdb/mapreduce/pull/268/files#r28416275>.
>